javalib/src/main/scala/java/lang/Character.scala

```diff
@@ -1662,429 +1963,467 @@ object Character {
-    BLOCKS_BY_NAME.update("BYZANTINE_MUSICAL_SYMBOLS",
-                          BYZANTINE_MUSICAL_SYMBOLS)
-    BLOCKS_BY_NAME.update("Musical Symbols", MUSICAL_SYMBOLS)
-    BLOCKS_BY_NAME.update("MusicalSymbols", MUSICAL_SYMBOLS)
-    BLOCKS_BY_NAME.update("MUSICAL_SYMBOLS", MUSICAL_SYMBOLS)
-    BLOCKS_BY_NAME.update("Tai Xuan Jing Symbols", TAI_XUAN_JING_SYMBOLS)
-    BLOCKS_BY_NAME.update("TaiXuanJingSymbols", TAI_XUAN_JING_SYMBOLS)
-    BLOCKS_BY_NAME.update("TAI_XUAN_JING_SYMBOLS", TAI_XUAN_JING_SYMBOLS)
-    BLOCKS_BY_NAME.update("Mathematical Alphanumeric Symbols",
-                          MATHEMATICAL_ALPHANUMERIC_SYMBOLS)
-    BLOCKS_BY_NAME.update("MathematicalAlphanumericSymbols",
-                          MATHEMATICAL_ALPHANUMERIC_SYMBOLS)
-    BLOCKS_BY_NAME.update("MATHEMATICAL_ALPHANUMERIC_SYMBOLS",
-                          MATHEMATICAL_ALPHANUMERIC_SYMBOLS)
-    BLOCKS_BY_NAME.update("CJK Unified Ideographs Extension B",
-                          CJK_UNIFIED_IDEOGRAPHS_EXTENSION_B)
-    BLOCKS_BY_NAME.update("CJKUnifiedIdeographsExtensionB",
-                          CJK_UNIFIED_IDEOGRAPHS_EXTENSION_B)
-    BLOCKS_BY_NAME.update("CJK_UNIFIED_IDEOGRAPHS_EXTENSION_B",
-                          CJK_UNIFIED_IDEOGRAPHS_EXTENSION_B)
-    BLOCKS_BY_NAME.update("CJK Compatibility Ideographs Supplement",
-                          CJK_COMPATIBILITY_IDEOGRAPHS_SUPPLEMENT)
-    BLOCKS_BY_NAME.update("CJKCompatibilityIdeographsSupplement",
-                          CJK_COMPATIBILITY_IDEOGRAPHS_SUPPLEMENT)
-    BLOCKS_BY_NAME.update("CJK_COMPATIBILITY_IDEOGRAPHS_SUPPLEMENT",
-                          CJK_COMPATIBILITY_IDEOGRAPHS_SUPPLEMENT)
-    BLOCKS_BY_NAME.update("Tags", TAGS)
-    BLOCKS_BY_NAME.update("Variation Selectors Supplement",
-                          VARIATION_SELECTORS_SUPPLEMENT)
-    BLOCKS_BY_NAME.update("VariationSelectorsSupplement",
-                          VARIATION_SELECTORS_SUPPLEMENT)
-    BLOCKS_BY_NAME.update("VARIATION_SELECTORS_SUPPLEMENT",
-                          VARIATION_SELECTORS_SUPPLEMENT)
-    BLOCKS_BY_NAME.update("Supplementary Private Use Area-A",
-                          SUPPLEMENTARY_PRIVATE_USE_AREA_A)
-    BLOCKS_BY_NAME.update("SupplementaryPrivateUseArea-A",
-                          SUPPLEMENTARY_PRIVATE_USE_AREA_A)
-    BLOCKS_BY_NAME.update("SUPPLEMENTARY_PRIVATE_USE_AREA_A",
-                          SUPPLEMENTARY_PRIVATE_USE_AREA_A)
-    BLOCKS_BY_NAME.update("Supplementary Private Use Area-B",
-                          SUPPLEMENTARY_PRIVATE_USE_AREA_B)
-    BLOCKS_BY_NAME.update("SupplementaryPrivateUseArea-B",
-                          SUPPLEMENTARY_PRIVATE_USE_AREA_B)
-    BLOCKS_BY_NAME.update("SUPPLEMENTARY_PRIVATE_USE_AREA_B",
-                          SUPPLEMENTARY_PRIVATE_USE_AREA_B)
+    import scala.Predef.ArrowAssoc
+    private val blocksByNormalizedName
+        : scala.collection.Map[String, UnicodeBlock] = scala.collection.Map(
```

Comment:
Again, baseline code uses scala map.  There was an effort in Scala.js to reduce/eliminate the use of scala.collection, at least in javalib, in favor of java collections
(which are a Royal Pain!).    

I think there has been no organized effort to do that yet in SN. Perhaps changing this can be delayed due to prior use in the baseline code.  I am not the one to make that
call but thought I would point the effort out.